### PR TITLE
Add qonnx

### PR DESCRIPTION
**Pull request by isuruf**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

cc @timkpaine

### CODE REVIEW

File: recipes/qonnx/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    - onnxruntime >=1.15.0
+    - sigtools >=4.0.1
+    - toposort >=1.7.0
+
+test:
+  imports:
+    - qonnx
+  commands:
+    # the package has a very tight protobuf pin
+    # but it protobuf is not used.
```
Comment (isuruf) [inline recipes/qonnx/meta.yaml:48]: Also needs https://github.com/conda-forge/clize-feedstock/pull/9 and https://github.com/conda-forge/sigtools-feedstock/pull/16 before pip check can be enabled.

**Comment by timkpaine** [comment]
confirmed

**Comment by isuruf** [comment]
@conda-forge-admin, lint

**Comment by isuruf** [comment]
Thanks

**Pull request merged**